packages/babel-parser/src/parser/expression.js

```diff
@@ -931,7 +935,7 @@ export default class ExpressionParser extends LValParser {
       elts.push(
         this.parseExprListItem(
           false,
-          possibleAsyncArrow ? new ExpressionErrors() : undefined,
+          refExpressionErrors || new ExpressionErrors(),
```

Comment:
Do we need to always create `new ExpressionErrors()` if `refExpressionErrors` is not defined, or can we do something like this?

```suggestion
          refExpressionErrors || (possibleAsyncArrow ? new ExpressionErrors() : undefined),
```

Or actually, can we just always forward `refExpressionErrors` since we now pass a `new ExpressionErrors()` instance when calling `parseCallExpressionArguments`?

```suggestion
          refExpressionErrors,
```

Reply:
`refExpressionErrors` is provided if and only if `possibleAsyncArrow` is `true`. Guess we can simplify the interface of `parseCallExpressionArguments` and replace the `possibleAsyncArrow` argument.

Reply:
Hey Guys 👋 

Thanks a lot for your review.

@JLHwung could you explain a bit more when you said: `replace the possibleAsyncArrow argument"` ?

I can modify it really quick when we'll aligned ^^

Thanks a lot guys ✋

Reply:
That's a good question. If the proposition that `possibleAsyncArrow` is `true` is equivalent to the proposition that `refExpressionErrors !== undefined`, then we can replace `possibleAsyncArrow` by `refExpressionErrors !== undefined` so we can get rid of one parameter, which is more performant.

Claim: I didn't actually test it. It is up to you to discover whether it works. 😄

Reply:
I finally deleted `possibleAsyncArrow` and as `parseCallExpressionArguments` is called somewhere without `refExpressionErrors` so I let: `refExpressionErrors || new ExpressionErrors(),`.